pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -78,24 +79,58 @@ type Check struct {
 	planEmitted                             *cache.Cache
 }
 
+func handleServiceCheck(c *Check, err error) {
+	sender, errSender := c.GetSender()
+	if errSender != nil {
+		log.Errorf("failed to get sender for service check %s", err)
+	}
+
+	message := ""
+	var status servicecheck.ServiceCheckStatus
+	if err == nil {
+		status = servicecheck.ServiceCheckOK
+	} else {
+		status = servicecheck.ServiceCheckCritical
+		message = fmt.Sprintf("%s", err)
+		log.Errorf("connect error %s", message)
```

Comment:
Couldn't this just be this?
```suggestion
		log.Errorf("failed to connect: %s", err)
```